[Workers] logpush changes (#6721)
cloudflare/cloudflare-docs: content/workers/platform/pricing.md

- (6,400,000 seconds) \* 128 MB / 1 GB = 800,000 GB-seconds
- (800,000 GB-s - included 400,000 GB-s) x $12.50 / 1,000,000 GB-s = $5.00

> ## Workers Trave Events Logpush

Reviewer: typo here

Workers Logpush is only available on the Workers Paid plan. 

{{<table-wrap>}}

|                             | Paid plan                          |
| --------------------------- | ---------------------------------- |
| Requests <sup>1</sup>       | 10 million / month, +$0.05/million |

{{</table-wrap>}}

> <sup>1</sup> Workers Logpush charges for request logs that make it to your end destination after filtering or sampling has been applied. 

Reviewer 2: ```suggestion
<sup>1</sup> Workers Logpush charges for request logs that reach your end destination after applying filtering or sampling. 
```
